jekyll/_cci2/create-project.adoc

```diff
@@ -12,40 +12,41 @@ contentTags:
 :icons: font
 :experimental:
 
-This guide gets you started with creating a project in CircleCI.
+This guide gets you started with creating a new project in CircleCI.
 
+[#prerequisites]
+== Prerequisites
 
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A code repository you want to build on CircleCI.
 
-[#step-one-link-your-vcs-with-circleci]
-== Step One: Link your VCS with CircleCI
+== Build a new project on CircleCI
 
-If you have not done so already, <<first-steps#,sign up with CircleCI>> and select your Version Control System (VCS). You can also sign up with email.
+The authentication method used to set up your CircleCI account determins the definition of "project" in CircleCI, as well as the permissions management processes available to you:
 
-A CircleCI project must be linked with an existing repo in your VCS. Make sure you have a repo and that you have authorized CircleCI to access it (in GitHub, you have the option to block CircleCI from accessing your private repositories).
+* If you authenticated using **GitHub OAuth**, or **Bitbucket**, a "project" in CircleCI maps directly to a "repository" in GitHub. Follow steps to <<set-up-a-project>> below.
+* If you authenticated through **GitLab**, a "project" within CircleCI is a standalone entity that you create and then associate with a code repository. Follow steps to <<create-a-project>> below.
```

Comment:
Do we need to also mention GitHub apps here? We do it in the "Create a project" section

Reply:
Done, I had forgotten to push this up but sone now